data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -222,12 +225,38 @@ def _create_tile_data(
             axis=0,
         )
 
+        logger.info("Adding fields for island areas and Puerto Rico")
```

Comment:
This is kind of unwieldy, but it should be reasonably quick and clear? Any thoughts here? The other Q I have is this leaves the tiles with two "not short" names -- Region and Thresholds. Am wondering if this is a problem?

Reply:
There's also a tradeoff here between creating these columns here, once we've narrowed the set of columns in the df, and creating the columns earlier such that these field names can be used in the dictionary. I happen to think (personally) that this should happen as late in the pipeline as possible, but I don't feel super strongly, so overrule me away!

Reply:
Discussion moved to slack on the second comment.

Reply:
Okay so I thought a little more about this and (for now). 

Yesterday, you mentioned that your instinct was to have this upstream, for the following reasons:
__pros:__
- makes this data available in the “all the things” file, keeping the spirt of that file having “all the things”
- researchers using the “big” file would be able to run analyses using the total number of thresholds
- we might want the total # of thresholds in the download xslx eventually

__cons:__
- might be a confusing field to see for people looking at the file
- weird one-off fixes should generally be targeted at specific areas of the codebase, not perpetuated through the whole thing

Then we talked a little bit about splitting this up -- having a threshold count available upstream and the rest down by the tiles. The issue there is the logic gets split up and it could be more confusing. 

I'm leaning towards keeping this where it is. I think deeming this a UI-only change makes sense because all of the information contained in this step is contained in other columns. What I mean is that as a data scientist, I would prefer documentation (even an additional table) that explains how the thresholds were computed, and NaNs in the actual file with the thresholds when the data for that threshold is missing. This could just be weird quirks about me, though -- but I think researchers who wanted to do analysis that included number of thresholds would already have that information. 

This is just my gut, though --  ultimately it's your call

Reply:
I endorse your proposal, your reasoning makes a lot of sense to me! (And though I have longer tenure, my vote counts the same as yours 😄 .) 

Let's do that for now. We can always refactor later if we learn more and change our minds.